src/index.tsx

```diff
@@ -57,6 +57,7 @@ export const defaultState: IMainState = {
         to: 'isv',
     },
     interfaceLang: getPreferredLanguage(),
+    colorTheme: 'light',
```

Comment:
I think you could use a smart default:

`window.matchMedia('(prefers-color-scheme: dark)').matches`

instead of using just 'light' theme.